src/core/config/mac.h

```diff
@@ -489,16 +489,27 @@
 #endif
 
 /**
- * @def OPENTHREAD_CONFIG_CSL_MIN_RECEIVE_ON
+ * @def OPENTHREAD_CONFIG_MIN_RECEIVE_ON_AHEAD
  *
- * The minimum CSL receive window (in microseconds) required to receive an IEEE 802.15.4 frame.
- * - Maximum frame size with preamble: 6*2+127*2 symbols
- * - AIFS: 12 symbols
- * - Maximum ACK size with preamble: 6*2+33*2 symbols
+ * The minimum time (in microseconds) before the MHR start that the radio should be in receive state and ready to
+ * properly receive in order to properly receive any IEEE 802.15.4 frame. Defaults to the duration of SHR + PHR.
  *
  */
-#ifndef OPENTHREAD_CONFIG_CSL_MIN_RECEIVE_ON
-#define OPENTHREAD_CONFIG_CSL_MIN_RECEIVE_ON 356 * 16
+#ifndef OPENTHREAD_CONFIG_MIN_RECEIVE_ON_AHEAD
+#define OPENTHREAD_CONFIG_MIN_RECEIVE_ON_AHEAD (6 * OT_RADIO_SYMBOL_TIME * OT_RADIO_SYMBOLS_PER_OCTET)
```

Comment:
Need to add the include header `openthread/platform/radio.h`?

Reply:
It seems it doesn't work for Ninja builds:

```
ERROR at //src/core/config/mac.h:38:11: Include not allowed.
#include <openthread/platform/radio.h>
```

Reply:
I can build it with `THREAD_VERSION=1.3 INTER_OP=0 OT_OPTIONS='-DOT_MTD=OFF' ./script/test build`. What command did you use?

Reply:
Okay, I went back to use numerical value to avoid configuration inter-dependency issues. Would this be okay to be merged for now?

Reply:
It's ok. Let's change it later.